SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -52,6 +52,7 @@ void USpatialGDKEditorSettings::PostInitProperties()
 	PlayInSettings->SaveConfig();
 
 	SetRuntimeWorkerTypes();
+	SetLevelEditorPlaySettingsWorkerTypes();
```

Comment:
I think this also needs to be called every time there's a change to the LaunchConfigDesc property.